### DISCUSSION TITLE
I can't see the Tab component in the sketch library

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#523] [General] [answered]
Hi! I am trying to find the tab component in the sketch library but can't find it anywhere. What am I missing?

**Comment by kate-porter** [comment, marked as answer]
Hey @pguptaDS, we are working on adding tabs to the Sketch library in our current sprint. With our new "alpha component" release model, some components will be released in Paste before they are available in the Sketch libraries (just look for the "Using alpha components in your product" callout at the top of the docs pages for these components).

If you need the tabs component ASAP, you can either use the legacy tabs in Sketch, or create them from scratch; just be sure to let your engineering team know to use Paste tabs. Or, if you hold out for just another day or so, the new tabs will be available for you in Sketch.

**Reply by pguptaDS** [reply]
Ah ok! I can wait. Thanks, Katie!

**Reply by kate-porter** [reply]
Tabs were just merged into the Paste components library. :)

You'll notice they look different than what is currently on the Paste docs site. The tabs in Paste will be updated to match the designs very soon.

**Comment by pguptaDS** [comment]
Oh great. I will check it out. Thanks, Katie!
